### PR TITLE
Better explained

### PR DESCRIPTION
**Pull request by spacephoton**
- Quick explanation updates (double check for correctness)
- Links are blue for consistent look

### CODE REVIEW

File: src/charts/TypeDistributionChart.tsx [hunk 1]
```diff
@@ -107,7 +107,7 @@ export const TypeDistributionChart = React.memo(
       : [];
 
     return ready && data.length > 0 && currentData ? (
-      <ChartAndMetrics metrics={metrics} title='Proportion of the variant by age (estimated)'>
+      <ChartAndMetrics metrics={metrics} title='Proportion of the variant by age group (estimate)'>
```
Comment (corneliusroemer) [inline src/charts/TypeDistributionChart.tsx]:
```suggestion
      <ChartAndMetrics metrics={metrics} title='Proportion of the variant per age group in the selected time frame (estimate)'>
```

File: src/pages/FocusPage.tsx [hunk 2]
```diff
@@ -224,16 +224,14 @@ export const FocusPage = ({
...
-                title='Estimation of the current advantage'
+                title='Transmission advantage'
                 toolbar={deepFocusButtons.chen2021Fitness}
-                description='
-              If variants spread pre-dominantly by local transmission across demographic groups, this
-               estimate reflects the transmission advantage of the focal variant. Importantly, the transmission
-               advantage estimate reflects the advantage compared to co-circulating strains. Thus, as new variants
-                spread, the advantage of the focal variant may decrease. When absolute numbers of a variant are
+                description={`
+              How easily this variant gets transmitted. This variant's so-called "advantage" may decrease as other more transmittable ones spread.
```
Comment (corneliusroemer) [inline src/pages/FocusPage.tsx]:
```suggestion
              How much faster or slower this variant is transmitted compared to the other variants present in this location. This variant's so-called "advantage" may decrease as other more transmittable ones spread.
```

**Review by corneliusroemer — COMMENTED** [review]
Made two suggestions for further improvement

File: src/models/chen2021Fitness/Chen2021ProportionPlot.tsx [hunk 3]
```diff
@@ -65,7 +65,7 @@ export const Chen2021ProportionPlot = ({
 
   return (
     <>
-      <TitleWrapper id='graph_title'>Estimated proportion through time*</TitleWrapper>
+      <TitleWrapper id='graph_title'>Proportion through time* (estimate)</TitleWrapper>
```
Comment (chaoran-chen) [inline src/models/chen2021Fitness/Chen2021ProportionPlot.tsx]: I would prefer having "estimated" as the first word.

File: src/pages/FocusPage.tsx [hunk 4]
```diff
@@ -224,16 +224,14 @@ export const FocusPage = ({
...
+              This assumes variants spread pre-dominantly by local transmission across demographic groups. When absolute numbers of a variant are
                 low, the advantage may merely reflect the current importance of introductions from abroad or
                  the variant spreading in a particular demographic group. In this case, the estimate does not
-                  provide information on the transmission advantage.'
+                  provide information on the transmission advantage.`}
```
Comment (chaoran-chen) [inline src/pages/FocusPage.tsx]: As discussed in our meeting, let's not change the description of this plot for now :)

**Review by chaoran-chen — CHANGES_REQUESTED** [review]

**Pull request merged**